lib/MusicBrainz/Server/Data/Statistics.pm

```diff
@@ -398,6 +398,31 @@ my %stats = (
             };
         },
     },
+    'count.event.type.typename.has_eventart' => {
+        DESC => 'Count of event with event art, by event type',
+        CALC => sub {
+            my ($self, $sql) = @_;
+
+            my $data = $sql->select_list_of_lists(
+                q{SELECT
+                   coalesce(event_type.name, 'null'),
+                   count(DISTINCT event_art.event)
+                 FROM event_art_archive.event_art
+                 JOIN event ON event.id = event_art.event
+                 FULL OUTER JOIN event_type
+                   ON event_type.id = event.type
+                 GROUP BY coalesce(event_type.name, 'null')},
+            );
+
+            my %dist = map { @$_ } @$data;
+
+            +{
+                map {
+                    'count.event.type.'.$_. '.has_eventart' => $dist{$_}
```

Comment:
You can ignore this since the file is riddled with stuff like this, but would be nice to change them all at some point. :P
```suggestion
                    "count.event.type.$_.has_eventart" => $dist{$_}
```

Reply:
Pushed a commit for this.